lib/wallet.js

```diff
@@ -1,4 +1,7 @@
 module.exports = function(options) {
+  var EC = require('elliptic').ec;
+  var ec = new EC('secp256k1');
```

Comment:
@sublimator What's the status of the elliptic conversion? If we move to elliptic, then I guess it's ok to use elliptic here. But if we use elliptic, we should get rid of sjcl entirely soon.